cli/011-cli-multi-config.md

```diff
@@ -0,0 +1,74 @@
+- Start Date: 2021-08-22
+- RFC PR:
+- Issue:
+
+### Configuring multiple environments
+
+##### Problem
+
+Users may have multiple Temporal service environments (development, production, ..) with their own respective mTLS configs, namespaces, address etc. Current implementation of tctl config feature only supports configuring a single environment and doesn't help at quickly switching between multiple.
+
+##### Proposal
+
+To make it easy for users to switch between environments, allow configuring multiple instead of just one and provide command to switch between them.
+
+Syntax to activate environment configs:
+
+```shellsession
+$ tctl config use-environment <name>
+Active environment: <name>
+```
+
+If the environment doesn't yet exist in config, create it and set as active
+
+```shellsession
+$ tctl config use-environment <name>
```

Comment:
I suppose this can be added later or configured via an alias, but something short like `tctl use <environment-name>` could be nice to have out of the box.

Reply:
Good point, +1